src/components/Input/index.styled.ts

```diff
@@ -71,6 +72,10 @@ export const InputStyled = styled.input<IInputProps>`
     font-weight: 300;
     color: ${props => props.theme.colors.text.primary};
   }
+   
+  :disabled {
+    color: ${props => props.theme.colors.input.border.default};
```

Comment:
text color = border color
looks a bit illogically

Reply:
fixed in f5a9af1f762e68cc9b15c6f3a6c0ac0104ef4767